packages/blade/src/components/Dropdown/DropdownOverlay.native.tsx

```diff
@@ -48,6 +48,7 @@ const _DropdownOverlay = ({ children, testID }: DropdownOverlayProps): React.Rea
           position="absolute"
           width="100%"
           testID="dropdown-overlay"
+          elevation={bottomSheetAndDropdownGlue?.dropdownHasBottomSheet ? undefined : 'midRaised'}
```

Comment:
removed styled-components based elevation and used the new elevation prop Anurag introduced on Box